v8engine/src/main/java/cm/aptoide/pt/v8engine/view/account/store/ManageStoreFragment.java

```diff
@@ -111,20 +195,71 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
   }
 
   @Override public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+    super.onViewCreated(view, savedInstanceState);
 
-    header = (TextView) view.findViewById(R.id.create_store_header);
-    chooseStoreNameTitle = (TextView) view.findViewById(R.id.create_store_choose_name_title);
-    selectStoreImageButton = view.findViewById(R.id.create_store_image_action);
-    storeImage = (ImageView) view.findViewById(R.id.create_store_image);
-    storeName = (EditText) view.findViewById(R.id.create_store_name);
-    storeDescription = (EditText) view.findViewById(R.id.edit_store_description);
-    cancelChangesButton = (Button) view.findViewById(R.id.create_store_skip);
-    saveDataButton = (Button) view.findViewById(R.id.create_store_action);
-    themeSelectorView = (RecyclerView) view.findViewById(R.id.theme_selector);
+    bindViews(view);
+    setupViews();
+
+    final AccountPermissionProvider accountPermissionProvider =
+        new AccountPermissionProvider(((PermissionProvider) getActivity()));
+
+    final StoreManager storeManager =
+        ((V8Engine) getActivity().getApplicationContext()).getStoreManager();
+
+    final String packageName = (getActivity().getApplicationContext()).getPackageName();
+
+    final String fileProviderAuthority = Application.getConfiguration()
+        .getAppId() + ".provider";
+
+    final PhotoFileGenerator photoFileGenerator = new PhotoFileGenerator(getActivity(),
+        getActivity().getExternalFilesDir(Environment.DIRECTORY_PICTURES), fileProviderAuthority);
+
+    final CrashReport crashReport = CrashReport.getInstance();
+    final UriToPathResolver uriToPathResolver =
+        new UriToPathResolver(getActivity().getContentResolver(), crashReport);
+
+    final ImagePickerNavigator imagePickerNavigator =
+        new ImagePickerNavigator(getActivityNavigator());
+
+    final ImageValidator imageValidator =
+        new ImageValidator(ImageLoader.with(getActivity()), Schedulers.computation());
+
+    final ImagePickerPresenter imagePickerPresenter =
+        new ImagePickerPresenter(this, crashReport, accountPermissionProvider, photoFileGenerator,
+            imageValidator, AndroidSchedulers.mainThread(), uriToPathResolver,
+            imagePickerNavigator);
+
+    final ManageStoreNavigator manageStoreNavigator =
+        new ManageStoreNavigator(getFragmentNavigator(), goToHome);
+
+    final ManageStorePresenter presenter =
+        new ManageStorePresenter(this, crashReport, storeManager, getResources(), uriToPathResolver,
+            packageName, manageStoreNavigator);
+
+    attachPresenter(new CompositePresenter(Arrays.asList(imagePickerPresenter, presenter)), null);
+  }
+
+  @Override public void onDestroyView() {
+    dismissWaitProgressBar();
+    if (dialogFragment != null) {
+      dialogFragment.dismiss();
+      dialogFragment = null;
+    }
+    super.onDestroyView();
+  }
+
+  public void setupViews() {
```

Comment:
No. The view has its initial state, which in that case is set by the setupViews() method.